web/themes/custom/pineapple/templates/content/node--recipe--full.html.twig

```diff
@@ -102,17 +101,17 @@
     </div>
     <aside class="recipe-sidebar">
       <div class="recipe-author-container">
+        {% set author_image_url = bamboo_render_image_style(content.field_author|field_value|first['#node'].getFields().field_a_image.entity.field_media_image.entity.getFileUri(), 'thumbnail_115_x_115') %}
```

Comment:
Turn everything from content to node into a variable then you can reuse that variable across this file instead of writing that long path everytime you wanna reference the author node

Reply:
Done.